release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2631,6 +2631,8 @@ To avoid this issue, you can enable confidential VMs on an existing cluster by u
 
 * When you run hosted control planes for {product-title} on a bare-metal platform, if a worker node fails, another node is not automatically added to the hosted cluster, even when other agents are available. As a workaround, manually delete the machine that is associated with the failed worker node. (link:https://issues.redhat.com/browse/MGMT-15939[*MGMT-15939*])
 
+* Since the source catalog bundles an architecture specific opm binary, you should run the mirroring from that architecture. For instance if you are mirroring a ppc64le catalog, you should run oc-mirror from a system that runs on the ppc64le architecture. (link:https://issues.redhat.com/browse/OCPBUGS-22264[*OCPBUGS-22264*])
```

Comment:
Is "should" actually "must" for this issue?

```suggestion
* Since the source catalog bundles an architecture specific `opm` binary, you should run the mirroring from that architecture. For instance if you are mirroring a ppc64le catalog, you should run oc-mirror from a system that runs on the ppc64le architecture. (link:https://issues.redhat.com/browse/OCPBUGS-22264[*OCPBUGS-22264*])
```

Reply:
@jeana-redhat I have updated the PR as per above. Request you to check. Thanks.